stable/spark/values.yaml

```diff
@@ -8,31 +8,37 @@ Spark:
 
 Master:
   Name: master
-  Image: "k8s.gcr.io/spark"
-  ImageTag: "1.5.1_v3"
+  Image: "dbanda/spark2.4"
+  ImageTag: "v3"
   Replicas: 1
   Component: "spark-master"
   Cpu: "100m"
-  Memory: "512Mi"
+  Memory: "1024Mi"
   ServicePort: 7077
   ContainerPort: 7077
   # Set Master JVM memory. Default 1g
   # DaemonMemory: 1g
   ServiceType: LoadBalancer
 
+WorkerLB:
```

Comment:
What does this WorkerLB do and as a user why do I need it? Can you please add to the README.md

Reply:
The worker load balancer is in case you want to use your spark cluster as a distributed web server with Spark Serving:

aka.ms/spark has some links to more info on this project

Reply:
@mhamilton723 I must have missed some of the wider context here, because of my lack of experience with Spark. 

Am I right that this PR now repurposes the more general Apache Spark chart, to focus primarily on Microsoft Machine Learning for Apache Spark? 

If so, would it be better to add a section for MML, with a `mml-values.yaml` override file (for default image, etc)? The changes to this chart are not large, so I'm hard pressed to suggest it warrants a separate chart, thought that would also be an option.

Reply:
@scottrigby actually, this chart and the accompanying docker images have very little do with Microsoft Machine Learning for Spark (mmlspark) . The only components from mmlspark included here are a few tiny sample spark notebooks included on the zeppelin image https://github.com/Azure/mmlspark/tree/master/tools/helm/docker_images/zepplin/mmlsparkExamples. Other than that, the rest of the chart is pure vanilla Spark, Hadoop, Linux.

To summarize, the main contributions in this PR are:

1. Update the spark to 2.4.0, hadoop 3.1.1
2. Add an optional Apache Livy pod to allow job submission via REST api. We found this super helpful. As this was the cleanest way to submit jobs to spark from outside the k8s cluster.
3. Include dockerfiles for the images for transparency. As far as we know, there was no easy way to get the dockerfiles for the previous images (k8s.gcr.io/spark) this made it a little difficult to debug.

We also fix an issue where the Zeppelin notebook was not set up to use the existing spark master within the cluster. Other than that, we tried to keep the changes in the PR minimal. I hope that helps.